Source: Gentoo Bug #187422
Upstream: no
Reason: compliance with Exherbo paths
diff -ur beid-middleware-3.5.2-source-5800.orig/_src/eidmw/common/Config.cpp beid-middleware-3.5.2-source-5800/_src/eidmw/common/Config.cpp
--- beid-middleware-3.5.2-source-5800.orig/_src/eidmw/common/Config.cpp	2009-06-17 16:11:02.000000000 +0200
+++ beid-middleware-3.5.2-source-5800/_src/eidmw/common/Config.cpp	2009-08-06 17:15:45.000000000 +0200
@@ -54,7 +54,7 @@
   void CConfig::Init(){
 
     if (! bIsInitialized){
-      const std::wstring system_datafilePath = utilStringWiden(STRINGIFY(EIDMW_PREFIX)) + L"/etc/";
+      const std::wstring system_datafilePath = L"/etc/beid/";
       home_path = utilStringWiden(std::string(getenv("HOME")));
 #ifdef __APPLE__
       const std::wstring user_datafilePath = L"Library/Preferences/";
diff -ur beid-middleware-3.5.2-source-5800.orig/_src/eidmw/dialogs/dialogsQTsrv/main.cpp beid-middleware-3.5.2-source-5800/_src/eidmw/dialogs/dialogsQTsrv/main.cpp
--- beid-middleware-3.5.2-source-5800.orig/_src/eidmw/dialogs/dialogsQTsrv/main.cpp	2009-06-17 16:11:06.000000000 +0200
+++ beid-middleware-3.5.2-source-5800/_src/eidmw/dialogs/dialogsQTsrv/main.cpp	2009-08-06 17:56:39.000000000 +0200
@@ -51,7 +51,7 @@
 
 
 const static char csOverWrite[] = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
-std::string readableFilePath = "/usr/local/etc/beidgui.conf";
+std::string readableFilePath = "/etc/beid/beidgui.conf";
 
 DlgDisplayPinpadInfoArguments *oInfoData = NULL;
 dlgWndPinpadInfo *dlgInfo = NULL;
diff -ur beid-middleware-3.5.2-source-5800.orig/_src/eidmw/install/beidgui35.desktop beid-middleware-3.5.2-source-5800/_src/eidmw/install/beidgui35.desktop
--- beid-middleware-3.5.2-source-5800.orig/_src/eidmw/install/beidgui35.desktop	2008-12-11 13:54:44.000000000 +0100
+++ beid-middleware-3.5.2-source-5800/_src/eidmw/install/beidgui35.desktop	2009-08-06 17:00:35.000000000 +0200
@@ -7,10 +7,10 @@
 Comment=Display and administer your eID card
 Comment[nl]=Weergeven en beheren van uw eID kaart
 Comment[fr]=Affichage et gestion de votre carte eID
-Exec=/usr/local/bin/beidgui
+Exec=/usr/bin/beidgui
 Terminal=false
 MultipleArgs=false
 Type=Application
-Icon=/usr/share/icons/eid35.png
+Icon=/usr/share/pixmaps/eid35.png
 Categories=Application
 
